src/mkdocstrings/plugin.py

```diff
@@ -267,9 +273,13 @@ def on_post_build(self, **kwargs) -> None:  # noqa: W0613,R0201 (unused argument
         this hook.
 
         Arguments:
+            config: The MkDocs config object.
             kwargs: Additional arguments passed by MkDocs.
         """
         if self._handlers:
+            css_content = "\n".join(handler.renderer.extra_css for handler in self.handlers.seen_handlers)
+            write_file(css_content.encode("utf-8"), os.path.join(config["site_dir"], self.css_filename))
```

Comment:
So I believe this file will be written in the `site` directory by default? Then shouldn't `css_filename` be `assets/stylesheets/_mkdocstrings.css"` instead?

Reply:
Yes.

There isn't any definitively correct path to use, because the paths are based entirely on each theme's own preference. [Material theme](https://github.com/squidfunk/mkdocs-material/tree/master/material) happens to use the path `assets/stylesheets`, while [MkDocs' own themes](https://github.com/mkdocs/mkdocs/tree/master/mkdocs/themes/readthedocs) don't use `assets/`anything at all.

I figured that using just "assets" still achieves the goal of reducing clutter in the output for Material theme, while also not creating the whole `assets/stylesheets` structure just to house this single file when using any other theme.

I don't have a strong preference, you can just tell me any final decision.

Reply:
Ah yeah I forgot about the themes. Then `assets/_mkdocstrings.css` is perfectly fine. Thanks!